kubernetes/deployment-production.tmpl

```diff
@@ -20,15 +20,82 @@ spec:
           env:
             - name: RAILS_ENV
               value: production
-            - name: DOTENV_FILE
-              value: /run/secrets/environment
             - name: PORT
               value: "81"
+            - name: RAILS_SERVE_STATIC_FILES
+              value: 'true'
+            - name: RAILS_MAX_THREADS
+              value: '2'
+            - name: REDIS_URL
+              value: redis://caesar-prod.lh4hyv.ng.0001.use1.cache.amazonaws.com
```

Comment:
No real opinion over here but this seems like the only questionable thing to keep in plaintext/source control.

Reply:
i wasn't sure either but that should be in a isolated network with closed security group access rules as well. the staging url is visible as well. It could be handy to quicky identify which redis service this talks too?

@adammcmaster thoughts on the public redis URL?

Reply:
You're right – access is restricted in the security group, and the Redis instance doesn't have a public IP, so it'll be fine for the URL to be public.